alignak/acknowledge.py

```diff
@@ -68,6 +69,7 @@ class Acknowledge(AlignakObject):  # pylint: disable=R0903
         'end_time': None,
         'author': None,
         'comment': None,
+        'comment_id': StringProp(default='')
```

Comment:
to reference the comment added in the host/service when create the acknowledge, like this, when delete the acknowledge, we delete the comment.
For the StringProp, I added a string property like other alignak objects